File: ex1/ex.go
package main

import (
    "fmt"
    "os"
    "path/filepath"
    "time"
)

func main() {
    dir := "/var/log/myapp"
    cutoff := time.Now().Add(-7 * 24 * time.Hour)
    filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
        if err == nil && !info.IsDir() && info.ModTime().Before(cutoff) {
            os.Remove(path)
            fmt.Println("Deleted:", path)
        }
        return nil
    })
}

File: ex1/ex.py
import os, time

log_dir = "/var/log/myapp"
now = time.time()
for f in os.listdir(log_dir):
    file_path = os.path.join(log_dir, f)
    if os.path.isfile(file_path) and now - os.path.getmtime(file_path) > 7 * 86400:
        os.remove(file_path)
        print(f"Deleted {file_path}")